integration/internal/swarm/service.go

```diff
@@ -59,6 +59,18 @@ func NewSwarm(t *testing.T, testEnv *environment.Execution, ops ...func(*daemon.
 	return d
 }
 
+// NewSwarmWithOption creates a swarm daemon for testing with init option we specify
+func NewSwarmWithOption(t *testing.T, initReq swarmtypes.InitRequest, testEnv *environment.Execution, ops ...func(*daemon.Daemon)) *daemon.Daemon {
```

Comment:
I don't think this is required… We should update the `NewSwarm` function instead, with a `WithInitOption` *operation*.

Reply:
WithInit option is used for Damon start.  I need to use Swarm Init.  could you pls let me know if I my understanding is correct ?